maker-shihab/longevity-catalyst-client
# Create Datepicker component and use it in add project page

## Description
This pull request introduces a dynamic display of the year and month in the DatePicker component's custom header when the month is increased or decreased.

## Changes Made
- Updated the `renderCustomHeader` function in the `DatePickerInput` component to display the current year and month dynamically.
- Utilized the `date.getFullYear()` and `date.getMonth()` methods to fetch the current year and month from the selected date.
- Added an array `monthNames` to map the numerical representation of the month to its corresponding name for better readability.

## Reason for Changes
Enhancing the user experience by providing clear visibility of the selected year and month during navigation within the DatePicker component.

## Related Issue
This pull request is related to [insert issue number here], which discusses improving the UI of the DatePicker component.

## How to Test
1. Navigate to the component where the DatePicker is being used.
2. Verify that the DatePicker now displays the current year and month in the custom header.
3. Increase and decrease the month using the navigation buttons and observe the changes in the displayed year and month.

## Screenshots (if applicable)
[Attach relevant screenshots if changes are visible in the UI.](https://github.com/maker-shihab/longevity-catalyst-client/blob/alifur/public/assets/img/screenshorts/Screenshot%202023-11-01%20060140.png)

![Screenshot](https://github.com/maker-shihab/longevity-catalyst-client/blob/alifur/public/assets/img/screenshorts/Screenshot%202023-11-01%20060140.png)

## Checklist
- [ ] Automated tests have been added or updated to cover the changes.
- [ ] Code follows the style guidelines of the project.
